examples/grpc_dynamic_codegen/capitalize_client.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const opentelemetry = require('@opentelemetry/core');
+const config = require('./setup');
+
+/**
+ * The trace instance needs to be initialized first, if you want to enable
+ * automatic tracing for built-in plugins (gRPC in this case).
+ */
+config.setupTracerAndExporters('grpc-client-service');
+
+const path = require('path');
+const grpc = require('grpc');
+const protoLoader = require('@grpc/proto-loader');
+
+const tracer = opentelemetry.getTracer();
+
+const PROTO_PATH = path.join(__dirname, 'protos/defs.proto');
+const PROTO_OPTIONS = { keepCase: true, enums: String, defaults: true, oneofs: true };
+const definition = protoLoader.loadSync(PROTO_PATH, PROTO_OPTIONS);
+const rpcProto = grpc.loadPackageDefinition(definition).rpc;
+
+function main () {
+  const client = new rpcProto.Fetch('localhost:50051',
+    grpc.credentials.createInsecure());
+  const data = process.argv[2] || 'opentelemetry';
+  console.log('> ', data);
+
+  const span = tracer.startSpan('tutorialsClient.capitalize');
+  tracer.withSpan(span, () => {
+    client.capitalize({ data: Buffer.from(data) }, function (err, response) {
+      if (err) {
+        console.log('could not get grpc response');
+        return;
+      }
+      console.log('< ', response.data.toString('utf8'));
```

Comment:
What do you think about logging the traceId here as well? This way its easy to verify propagation without using a UI.